src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  const getSum = (number) => {
+    counter++;
+
+    if (counter < 3) {
+      return (x) => x + number;
+    } else {
+      if (counter % 2 === 0) {
+        return (x) => 'Bzzz... Error!';
+      } else {
+        return (x) => x + number;
+      }
```

Comment:
You can use logical && in the if condition to get rid of extra nesting

Reply:
Agree, you should avoid nesting. Try to solve it without nesting